### DISCUSSION TITLE
How could I only enable active health check when create upstream using admin api

### DISCUSSION THREAD

**Discussion by zhangliweixyz** [apache/apisix#7914] [Q&A] [answered]
I want to create a upstream using apisix built-in admin api, and I only give active check just like below, but it add passive check by default, so what can I do to only enable active health check.
![image](https://user-images.githubusercontent.com/70928799/190075953-8a84e5bf-bffc-4325-b09a-1105f85d28b0.png)

**Comment by tzssangglass** [comment, marked as answer]
fixed by: https://github.com/apache/apisix/pull/7850

**Reply by tokers** [reply]
@zhangliweixyz Try to use the latest APISIX branch, or you may have to wait the next release.

**Reply by zhangliweixyz** [reply]
Thanks，I know how to do it.
